geomesa-lambda/geomesa-lambda-datastore/src/test/scala/org/locationtech/geomesa/lambda/LambdaTestRunnerTest.scala

```diff
@@ -64,16 +64,18 @@ object LambdaTestRunnerTest {
     var zookeepers: String = _
 
     lazy val dsParams = Map(
-      "lambda.accumulo.connector"  -> MiniCluster.connector,
+      "lambda.accumulo.instance.id" -> MiniCluster.cluster.getInstanceName,
+      "lambda.accumulo.zookeepers"  -> MiniCluster.cluster.getZooKeepers,
+      "lambda.accumulo.user"        -> MiniCluster.Users.root.name,
+      "lambda.accumulo.password"    -> MiniCluster.Users.root.password,
       // note the table needs to be different to prevent testing errors
```

Comment:
Does the test table still need to be different? 

Does this show that a Lambda DS catalog and an Accumulo DS catalog may not be shared?  If so, is that documented?

Reply:
you can share an accumulo and lambda ds table - this is just meant for the unit tests, so that they don't interfere with each other.